config/StoredProcedures/EditUserByUsername

```diff
@@ -1,14 +0,0 @@
-DROP PROCEDURE IF EXISTS EditUserByUsername;
-
-DELIMITER $$
-CREATE PROCEDURE EditUserByUsername
-(IN _username varchar(255), IN _firstname varchar(255), IN _lastname varchar(255), IN _permissions int, IN _confirmation bit)
-
-BEGIN
-
-UPDATE Users
-SET FirstName = _firstname, LastName = _lastname, PermsID = _permissions, IsConfirmed = _confirmation
-WHERE Username = _username;
-END $$
-
-DELIMITER ;
```

Comment:
Correct, so if we want to deactivate we can, otherwise we leave the confirmation bit.